test/user/fetch-users.spec.ts

```diff
@@ -13,9 +13,24 @@ import { createAndAuthenticateUser } from '../../src/utils/create-and-authentica
 describe('Fetch Many Users', () => {
   let token: string
   let server: ApolloServer
-  const query = `query{
+  const query = `query {
     users {
-      name email birthDate id
+      users {
+        name email id birthDate
```

Comment:
```suggestion
        name
        email 
        id 
        birthDate
```

não fica melhor pra ler?

Reply:
> não fica melhor pra ler?

fica bem melhor, vou corrigir!